### DISCUSSION TITLE
Are You Using Lamby?

### DISCUSSION THREAD

**Discussion by metaskills** [customink/lamby#59] [Show and tell]
* At what company? Can you share?
* How are you using it? Lift & shift? New?
* What else?

**Comment by equivalent** [comment]
So far to test what  Rails on Lambda can/cannot do https://ror.serverless-ruby.org/ 

Created via [quick start](https://lamby.custominktech.com/docs/quick_start)

What worked: 

* webpack/webpacker
* Cloudflare as a custom domain (so that I have free https)
* StimulusJS, Turbo Frames (http://hotwire.dev/)  from webpacker
* DynamoDB setup for Rails Models via [Dynamoid gem](https://github.com/Dynamoid/dynamoid)
* Redis caching server (i'm using free redislabs cluster $0, So NOT via AWS ElasticCache)

What didn't work

so far nothing  I've needed

**Comment by equivalent** [comment]
https://www.serverless-ruby.org/  Simple posting board (inspired by rubyflow.com) for serverless ruby links/articles/projects... 

it's a Real app (not a testing app)  but work in progress 

technologies: DynamoDB, Github Oauth via Omniauth, Turbo & StimulusJS. It was generated from QuickStart guide
